typography-nutrition-label/index06.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+    <link href="./css/style06.css" rel="stylesheet">
+  </head>
+  <style>
```

Comment:
These <style> tags seem to be extra. Should they be removed?

Reply:
@beaucarnes, great catch! Thank you, I totally missed that.